prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,58 @@
+"""
+CP1404 Practical- 09
+Ashton Jack Stewart
+Taxi simulator
+"""
+
+
+from taxi import Taxi
+from silver_service_taxi import SilverServiceTaxi
+
+def main():
+    """Sets an initial set of values for the class and functions to use"""
+    print("Lets drive!")
+    taxis = [Taxi("Prius", 100),
+             SilverServiceTaxi("Limo", 100, 2),
+             SilverServiceTaxi("Hummer", 200, 4)]
+    menu = "q)uit, c)hoose taxi, d)rive"
+    current_taxi = "not chosen"
+    choice = ""
+    bill_to_date = 0
+
+    choose_action(taxis, menu, current_taxi, choice, bill_to_date)
+
+def choose_action(taxis, menu, current_taxi, choice, bill_to_date):
+    """Gives the user a set of choices"""
+    while choice != "q":
```

Comment:
Could refactor into main. 
There is a lot being passed into this singular function.
It affects readability.